locales/cy/translations.json

```diff
@@ -11,7 +11,10 @@
         "checkNameExistingLP" : "WELSH - You must check the name is not the same as an existing limited partnership on the <a href='#/'>Companies House register (link opens in a new tab)</a>. If it is, we will reject your application.",
         "checkNameExistingTrademark" : "WELSH - You should also use the <a href='https://www.gov.uk/search-for-trade mark' target='_blank'>trade mark search (link opens in a new tab)</a> to check if your name matches an existing trade mark. If it does, consider choosing a different name. Companies House will not carry out this check for you.",
         "nameEnding" : "WELSH - Which limited partnership name ending would you prefer to go on the public record?",
-        "nameEndingHint" : "WELSH - There's no difference between the name endings Limited Partnership, LP and L.P. except for how they look."
+        "nameEndingHint" : "WELSH - There's no difference between the name endings Limited Partnership, LP and L.P. except for how they look.",
+        "existingPartnershipHint" : "You must check the name is not the same as an existing partnership or company on the <a href='https://find-and-update.company-information.service.gov.uk/company-name-availability'>Companies House register (opens in a new tab)</a>. If it is, we’ll reject your application.",
```

Comment:
can you start the welsh text with "WELSH - " , it will help us to identify the welsh text when testing.

Reply:
These sentences are now in Welsh.